nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
@kozlovic I hear what you are saying. Maybe if we mark type, and document that only explicit servers (pre-configured) can contain credentials that might suffice. We would want to be consistent across all clients we support.